### ISSUE TITLE
[Accessibility][ScreenReader]: Going back to text elements with shift-tab reads the element you just tabbed from.

### ISSUE THREAD

**Issue by rp-dev** [microsoft/BotFramework-WebChat#3118]
## Screenshots

![image](https://user-images.githubusercontent.com/50723361/79809447-4f8eab00-8325-11ea-93b3-355e82512ac3.png)

## Version

v4

## Describe the bug

Actual: With JAWS ON, pressing keyboard tab to arrive at actionable links such as "Coronavirus/Covid-19 How to protect yourself and get care link" and other links,  JAWS announces "No" button for each link label. 
Expected:  JAWS should announce "Coronavirus/Covid-19 How to protect yourself and get care link"  (instead of "No" button)

## Additional context

[Bug]

**Comment by compulim** [comment]
This is fixed in #3287.
